connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/Plugins.java

```diff
@@ -602,7 +602,7 @@ private <U> U newVersionedPlugin(
                 // if the config specifies the class name, use it, otherwise use the default which we can get from config.getClass
                 String classOrAlias = config.originalsStrings().get(classPropertyName);
                 if (classOrAlias == null) {
-                    classOrAlias = config.getClass(classPropertyName).getName();
+                    classOrAlias = config.getClass(classPropertyName) == null ? null : config.getClass(classPropertyName).getName();
```

Comment:
Ah yeah, if there is neither an original string nor a default, getClass will return null. :+1: